openapi-connector-spec.yaml

```diff
@@ -421,6 +421,95 @@ paths:
         - BearerAuth: []
       tags:
         - groups
+  /groups/{group_id}/member-groups:
+    post:
+      description: Add a member group to the specified group
+      operationId: addGroupMemberGroup
+      parameters:
+        - description: The ID of the parent group
```

Comment:
@giulio-opal what nomenclature have we exposed so far, we call this a containing group instead of a parent group, but do we want to use the language with customers?

Reply:
I think we want to maintain the same nomenclature for customer facing apis as well. We haven't fully shipped this feature yet, so we're pretty flexible on what we can choose